src/main/java/pocketpal/frontend/constants/ParserConstants.java

```diff
@@ -0,0 +1,52 @@
+package pocketpal.frontend.constants;
+
+import java.util.regex.Pattern;
+
+public class ParserConstants {
```

Comment:
Perhaps this class could have the `final` keyword since it is not meant to be inherited, and have a `private` constructor since it is not meant to be instantiated?